README.md

```diff
@@ -57,6 +53,30 @@ EOF
 oc create -f my-registry-secret.yml
 ```
 
+#### Red Hat Catalog Imagestreams
+
+The pipelines must pull the parent index images through the internal OpenShift
+registry to take advantage of the built-in credentials for Red Hat's terms-based
+registry. There is a one time configuration for each desired distribution catalog.
```

Comment:
That is just my opinion that you can disagree with, but I think it's hard to understand the purpose of following actions from the first sentence.

Reply:
I added some more detail. Does that help?

Reply:
Yes, thank you! That's great. There is just a small typo there: 
`The saves the user`

Reply:
Fixed.